src/components/tekstomrade/parser/rules.ts

```diff
@@ -98,3 +114,5 @@ export const LinkRule: Rule = {
         };
     }
 };
+
+export const BaseRules = [LinkRule, LinebreakRule, ParagraphRule];
```

Comment:
Denne virker litt random for min del, e.g verdiene er noe som passer for akkurat våres bruk og ikke noe som nødvendigvis noe som kan generaliseres?